CMakeLists.txt

```diff
@@ -55,6 +61,50 @@ endif()
 # IREE-specific CMake configuration
 #-------------------------------------------------------------------------------
 
+# List of all backends to be built by default:
+set(IREE_ALL_BACKENDS
+  LLVM
+  Vulkan
+  VMLA
+)
+
+
+if( IREE_HAL_BACKENDS_TO_BUILD STREQUAL "all" )
+  set( IREE_HAL_BACKENDS_TO_BUILD ${IREE_ALL_BACKENDS} )
+endif()
+message(STATUS "Building HAL backends ${IREE_HAL_BACKENDS_TO_BUILD}")
+
+# Default every IREE_HAL_BACKEND_* to OFF
+foreach(_backend ${IREE_HAL_BACKENDS_TO_BUILD})
```

Comment:
Shouldn't this be looping of IREE_ALL_BACKENDS?

Reply:
Yes! Thanks for catching this. Got lost when splitting into HAL and compiler specific variables.